sdk/communication/azure-communication-callautomation/tests/call_automation_automated_live_test_base.py

```diff
@@ -0,0 +1,94 @@
+import os
+import json
+import time
+from datetime import datetime, timedelta
+from typing import Any, Dict, Optional, Type
+
+import requests
+from azure.communication.callautomation import CallAutomationEventParser
+from azure.servicebus import ServiceBusClient
+
+from _shared.asynctestcase import AsyncCommunicationTestCase
+
+
+class CallAutomationAutomatedLiveTestBase(AsyncCommunicationTestCase):
+    def __init__(self, method_name, *args, **kwargs):
+        self.servicebus_connection_str = os.environ.get('SERVICEBUS_STRING', 'Endpoint=sb://REDACTED.servicebus.windows.net/;SharedAccessKeyName=REDACTED;SharedAccessKey=REDACTEDu8EtZn87JJY=')
```

Comment:
nit: this last part of the sharedkey, was this supposed to be like this or the whole thing should be redacrted?